gui-apps/lavalauncher/lavalauncher-1.7.1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="simple launcher for wayland"
+HOMEPAGE="https://git.sr.ht/~leon_plickat/lavalauncher"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.sr.ht/~leon_plickat/lavalauncher"
+else
+	SRC_URI="https://git.sr.ht/~leon_plickat/lavalauncher/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/${PN}-v${PV}"
+	KEYWORDS="~amd64 ~arm64 ~x86"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+
```

Comment:
man? (https://git.sr.ht/~leon_plickat/lavalauncher/tree/master/meson.build)

Reply:
same with the scdoc, I think I will wait for upstream to patch first :smile_cat:

Reply:
done!